src/server/multiplayer/MultiplayerServerState.ts

```diff
@@ -54,10 +54,10 @@ export default class MultiplayerServerState {
   // Returns the username to use for the given player.
   // If fallbackToClientID is true, falls back to the client ID if there is no
   // username set, otherwise returns null.
-  getClientUsername = (clientID: m.ClientID, fallbackToClientID = true): string => (
+  getClientUsername = (clientID: m.ClientID, fallbackToClientID = true): string | null => (
```

Comment:
What do you think about using some implementation of optional to clean up the typing?
like: https://www.npmjs.com/package/optional-js

Reply:
Ooh, that's a good idea. I'll merge as-is for now since this would be a pretty major change, but I made a ticket – #733.